src/azure/Templates/AzureModelIndexTemplate.cshtml

```diff
@@ -27,4 +31,14 @@ export { BaseResource, CloudError };
 {
 @EmptyLine
 @:@(Include(new EnumTemplate(), model))
+}
+
+@foreach (var method in Model.MethodsWithHeaders) {
+@EmptyLine
+<text>
+/**
+@WrapComment(" * ", $"Contains response data for the {method.Name} operation.")
+ */
+</text>
+@:export interface @(Regex.Replace(method.ReturnType.Headers.Name, "Headers$", "Response")) extends msRest.HttpOperationResponse<@(method.ReturnType.Body?.TSType(inModelsModule: true) ?? "void"), @(method.ReturnType.Headers.Name)> {};
```

Comment:
I'd recommend moving this line up inside the <text> element so that you don't need the `@:`.